app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeriesActivity.java

```diff
@@ -49,23 +47,22 @@ protected String getTrackedViewName() {
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        
-        setContentView(R.layout.activity_special);
-
-        Toolbar toolbar = getActionBarToolbar();
-        toolbar.setTitle(R.string.devfest);
-
         mTaggedEventSeries = getIntent().getParcelableExtra(Const.EXTRA_TAGGED_EVENT);
         if (mTaggedEventSeries == null) {
-            throw new IllegalArgumentException("Special Event must be provided with " 
+            throw new IllegalArgumentException("Special Event must be provided with "
                     + Const.EXTRA_TAGGED_EVENT + " key as an Intent extra.");
         }
+        setTheme(mTaggedEventSeries.getColorResPrimary());
+        super.onCreate(savedInstanceState);
         
-        getSupportActionBar().setLogo(mTaggedEventSeries.getLogoResId());
+        setContentView(R.layout.activity_special);
 
-        mDescription.setText(mTaggedEventSeries.getDescriptionResId());
+        Toolbar toolbar = getActionBarToolbar();
 
+        toolbar.setTitle(mTaggedEventSeries.getTitleResId());
```

Comment:
I don't think so, the background is the colorPrimary attribute and I am changing that attribute in this Activity.